tools/converter.py

```diff
@@ -159,6 +159,8 @@ def write_test(self, filename_test, run=False):
 
         spec_str = f"import os, pytest \nfrom pathlib import Path\n"
         spec_str += f"from ..{self.interface_name.lower()} import {self.interface_name} \n\n"
+        if run:
+            spec_str += "@pytest.mark.skipif('FSLDIR' not in os.environ, reason='no FSL found')\n"
```

Comment:
If we use `xfail`, we can make sure that the reason it's failing is because the executable is not found, rather than some other issue that should be a bug...

```suggestion
            spec_str += "@pytest.mark.xfail('FSLDIR' not in os.environ, reason='no FSL found', raises=FileNotFoundError)\n"
```

Reply:
why do you want to change to `xfail`?

Reply:
not sure, how you would be checking it. it will just `xfail`..

Reply:
And when we add coverage, it will probably exercise more code.

Reply:
yes, I missed the `raises` part, will change!